spec/helpers.lua

```diff
@@ -39,18 +55,14 @@ function compare(func, number)
     return tostring(number), number == cnumber
 end
 
-local broker_list_plain = {
-	{ host = "broker", port = 9092 },
-}
 
--- define topics, keys and messages etc.
-TEST_TOPIC = "test"
-TEST_TOPIC_1 = "test1"
-KEY = "key"
-MESSAGE = "message"
-CERT = cert
-PRIV_KEY = priv_key
-BROKER_LIST = broker_list_plain
+-- Create topics before running the tests
+function create_topics()
```

Comment:
I'm not sure if that's a sane approach. There  is a bit of timing involved when Kafka creates the topics. 

We should consider adding a separate script before a test execution that creates test topics

Reply:
The current approach is very bad.. deferring a proper solution to later stages